django/contrib/admin/decorators.py

```diff
@@ -1,3 +1,63 @@
+def action_method(*, permissions=None, description=None):
+    """
+    Conveniently add attributes to an action method::
+
+        @admin.action_method(
+            permissions=['publish'],
+            description='Mark selected stories as published',
+        )
+        def make_published(self, request, queryset):
+            queryset.update(status='p')
+
+    This is equivalent to setting the attributes on the method directly::
+
+        def make_published(self, request, queryset):
+            queryset.update(status='p')
+        make_published.allowed_permissions = ['publish']
+        make_published.short_description = 'Mark selected stories as published'
+    """
+    def decorator(f):
```

Comment:
Not sure the style guide says it but I prefer variable names >1 character when they're used across multiple lines, so `func` here?

Reply:
Fair enough.